### ISSUE TITLE
Trigger Doesn't Output Analog Value

### ISSUE THREAD

**Issue by 0-ks** [kozec/sc-controller#739]
Using the default XBox Controller Profile, when I connect my DS4, it's not possible to emulate Analog trigger values.

As you can see bellow, User 0 is the emulated device by SC Controller and the User 1 is the actual input of the DS4 in Wine Control Panel:
![sc-controller-trigger-bug](https://github.com/user-attachments/assets/e4b7cbb7-ecb6-4151-9510-5fc2f69b17fa)

Here is the output of the emulated device in EVTEST to confirm the issue is not related to Wine:
![sc-controller-trigger-bug-evdev](https://github.com/user-attachments/assets/f7b399cb-41b4-4444-bb47-a174035988c3)

**Comment by 0-ks** [comment]
After further investigation, I found that changing the Analog Output Range Start and End to 128 and 129 resolves the problem, so it seems like the default range of 0 to 255 is causing the problem for some reason.

As a workaround, you can try setting:
`axis(Axes.ABS_Z, 128, 129)` for the Left Trigger Action
and
`axis(Axes.ABS_RZ, 128, 129)` for the Right Trigger Action
until the bug is fixed.

**Comment by C0rn3j** [comment]
If you can repro it on the [Python 3 fork](https://github.com/C0rn3j/sc-controller), feel free to report this bug there too.

**Comment by 0-ks** [comment]
> If you can repro it on the [Python 3 fork](https://github.com/C0rn3j/sc-controller), feel free to report this bug there too.

Yes, Unfortunately the bug is also present in the latest release of the Python 3 fork,

I will report [this on the Python 3 fork repo](https://github.com/C0rn3j/sc-controller/issues/21#issue-2528302268) as well.

**Comment by C0rn3j** [comment]
Fixed in my fork, it was a wrong if check.

https://github.com/C0rn3j/sc-controller/commit/ead96de94dba2609569e7359083cef2f37f8cc8c
